packages/@tinacms/react-forms/src/CreateContentMenu.tsx

```diff
@@ -116,44 +115,14 @@ const FormModal = ({ plugin, close }: any) => {
     [close, cms, plugin]
   )
 
-  const [busy, setBusy] = React.useState(false)
-
   return (
     <Modal>
-      <FormBuilder form={form}>
-        {({ handleSubmit, invalid }) => {
-          const awaitedHandleSubmit = () => {
-            setBusy(true)
-            handleSubmit()?.finally(() => {
-              setBusy(false)
-            })
-          }
-          return (
-            <ModalPopup>
-              <ModalHeader close={close}>{plugin.name}</ModalHeader>
-              <ModalBody
-                onKeyPress={e =>
-                  e.charCode === 13 && !busy
-                    ? (awaitedHandleSubmit() as any)
-                    : null
-                }
-              >
-                <FieldsBuilder form={form} fields={form.fields} />
-              </ModalBody>
-              <ModalActions>
-                <Button onClick={close}>Cancel</Button>
-                <Button
-                  disabled={busy || invalid}
-                  onClick={awaitedHandleSubmit as any}
-                  primary
-                >
-                  Create
-                </Button>
-              </ModalActions>
-            </ModalPopup>
-          )
-        }}
-      </FormBuilder>
+      <ModalPopup>
+        <ModalHeader close={close}>{plugin.name}</ModalHeader>
+        <ModalBody>
+          <FormMaker form={form} />
```

Comment:
In a Zoom Meeting on Tuesday, June 8th, @dwalkr, @spbyrne, and myself decided that _every_ TinaForm should have a standard set of actions - specifically, Save/Submit and Reset.

For a while, TinaCMS has been mixing controls for the Form with controls for the Modal.  In an effort to unify our Forms, we're going to try and not have a dedicated "Close/Cancel" button and instead we'll rely on the 'X' (cross) in the top-right corner.

What this means is that Submit, Reset, and any additional `actions:[]`* should appear with a `Form` regardless of where it is rendered.

* Once we move `actions` to utilize the plugin system.